examples/package.json

```diff
@@ -9,10 +9,13 @@
     "dev": "vite",
     "build": "vite build",
     "preview": "vite preview",
-    "test:all": "echo 'No tests available' && exit 0"
+    "test:all": "playwright test"
```

Comment:
`test:all` command is intended to run all tests, exactly like in lib [package.json](https://github.com/ukorvl/lightweight-charts-react-components/blob/main/lib/package.json#L9)
So, he i would propose to add a separate command for e2e tests like `"test:e2e": "playwright test"` and reference it in `test:all`.